### ISSUE TITLE
Pipeline for CACAO annotations

### ISSUE THREAD

**Issue by jimhu-tamu** [geneontology/go-site#1216]
Based on discussion today with @kltm @vanaukenk related to #1078.

CACAO annotations are multi-species and are currently exported from GONUTS as a gpad file at 

https://gowiki.tamu.edu/gpad/CACAO.gpad.gz

We use gpad for two reasons
* Extra column allows us to save the student notes justifying the annotation
* We switched to using ECO for evidence as of Spring 2019

I believe @alexsign picks this up for import into Protein2GO but I can't tell if that stepis working yet.

cc @jolenerr

**Comment by alexsign** [comment]
@jimhu-tamu yes and no. The automated pipeline is not quite ready yet, but the set of annotations from https://gowiki.tamu.edu/gpad/CACAO.gpad.gz is loaded in our database. It will be able availabe in out release files next week as well as in the QuickGO.

**Comment by alexsign** [comment]
@jimhu-tamu the pipeline now in productions and annotations updated weekly

**Comment by pgaudet** [comment]
@jolenerr

**Comment by jrr-cpt** [comment]
Thanks @alexsign @pgaudet , I did see the new annotations go out. We will be reviewing the CACAO process with @cross12tamu and @dsiegele
